x-pack/plugins/apm/server/lib/helpers/create_es_client/create_internal_es_client/index.ts

```diff
@@ -49,18 +58,32 @@ export function createInternalESClient({
     >(
       params: TSearchRequest
     ): Promise<ESSearchResponse<TDocument, TSearchRequest>> => {
-      return callEs('search', params);
+      return callEs({
+        operationName: 'search',
```

Comment:
Are these values defined as constants anywhere? It's clear as-is, but just wondering.

Reply:
No, it's used for logging only as well, the actual ES client call is separate from it.